cmd/telemetrygen/internal/logs/worker.go

```diff
@@ -26,6 +30,23 @@ type worker struct {
 	index          int             // worker index
 }
 
+var reRandom = regexp.MustCompile(`\${random:(\d+)-(\d+)}`)
```

Comment:
I would prefer to be more specific here and call this `random_range` or something similar. I would also look into OTTL and see if that kind of syntax would work here. This is not a trivial problem and I would like to do it right